model/structs.go

```diff
@@ -0,0 +1,60 @@
+package model
+
+type ImplementationStatus int32
+
+const (
+	Waiting ImplementationStatus = iota
+	Ready
+	Done
+)
+
+type SourceSystem struct {
+	Id          int32
+	Name        string
+	Description string
+}
+
+type EventType struct {
+	Id             int64
+	Name           string
+	RecommendedUse string
```

Comment:
what is this *RecommendedUse* supposed to be? Same question for *PreciseUse*

Reply:
Will docu all.